sdk/elixir/lib/dagger/gen/container.ex

```diff
@@ -1213,7 +1213,7 @@ defmodule Dagger.Container do
   @doc """
   Retrieves this container with an unset working directory.
 
-  Should default to "/".
+  Should default to \"/\".
```

Comment:
Do `"` need to be escaped in triple quotes? I get that `\` should be escaped.

Reply:
It is not necessary but need it can so I make consistent to both single and triple quote string.

Reply:
:thinking: it depends on what's common in elixir, but IMO we shouldn't be unnecessarily adding quotes. Yes, there's a visual difference, but `\"` is a different value in a single quoted string vs a triple quoted string.

If we want to keep it consistent, we could always use a triply-quoted string if there's a `"` in the string.

Reply:
Using triple quote for all of documentation would be a great idea. 

Fixing...